src/ast/ExecutionOrder.h

```diff
@@ -10,15 +10,14 @@
  *
  * @file ExecutionOrder.h
  *
- * Defines AST Clauses
+ * Defines the execution order class
  *
  ***********************************************************************/
 
 #pragma once
 
+#include "Node.h"
 #include "SrcLocation.h"
-#include "ast/Node.h"
-#include "utility/StreamUtil.h"
```

Comment:
This is needed for `join`

Reply:
I will add this - I noticed the dependency too late.